src/wallet/rpcwallet.cpp

```diff
@@ -2262,38 +2262,107 @@ UniValue getstakesplitthreshold(const UniValue& params, bool fHelp)
     return int(pwalletMain->nStakeSplitThreshold);
 }
 
+UniValue getautocombineinfo(const UniValue& params, bool fHelp)
+{
+    if (fHelp || params.size() != 0)
+        throw runtime_error(
+            "getautocombineinfo\n"
+            "Returns the autocombinerewards settings\n"
+            "\nResult:\n"
+            "1. enabled   (string) The feature turned \"on\" or \"off\".\n"
+            "2. threshold (numeric) If enabled on, returns autocombine threshold.\n"
+            "3. frequency (variable) If enabled, returns frequency in blocks, or \"nextblock\" if one time. "
```

Comment:
We really should not be returning non-deterministic data types in RPC results. You can use a numeric type with static negative values to map to "nextblock" and "startup". A 4th return field can be added to output a human-readable string in these cases.

Reply:
Logically, handling static negative values would complicate the code during the autocombine; which isn't a place I would want to add more logic, since it's something that's better to be efficient.  However, I can work with  a human-readable string; which can explain the 0 vs. threshold, or 0 vs. frequency and the "disabled with frequency 0" special cases.

Reply:
As long as each JSON return field is only ever of a single deterministic data type